Competition2017/src/competition/subsystems/RobotSide.java

```diff
@@ -0,0 +1,6 @@
+package competition.subsystems;
```

Comment:
Should this go in the subsystems package? It isn't really a subsystem, so the parent `competition` might make more sense.